csuntechlab/iSTART
# STR-226 -- Implement Multi Quiz Assessment

# Description
Implemented multi quiz assessment for all the modules
  
# Testing
1. Visit `Alcohol, Marijuana, Tobacco, & Prescription Drugs` assessment portion.
2. Check the following conditionals and results for each...
> All selected *Yes* input
> All selected *No* input
> Randomly selected input
3. Ensure all responses are reasonable.
